lib/tools.js

```diff
@@ -82,26 +89,38 @@ function asJson (obj, msg, num, time) {
   // we need the child bindings added to the output first so instance logged
   // objects can take precedence when JSON.parse-ing the resulting log line
   data = data + chindings
+
+  if (hasObj === false) return data + end
+
+  const notHasOwnProperty = obj.hasOwnProperty === undefined
+
+  if (objError === true) {
+    data += ',"type":"Error","stack":' + stringify(obj.stack)
+  }
+
+  // if global serializer is set, call it first
+  if (serializers[wildcardGsym]) {
+    obj = serializers[wildcardGsym](obj)
+  }
+
+
+  if (serializersInObject(serializers, obj) === false) {
```

Comment:
this returns a boolean, there is no need for `=== false`.

Reply:
```
if (serializersInObject(serializers, obj) === false) {
```
is more explicit and easier to read than 
```
if (!serializersInObject(serializers, obj)) {
```

Explicit checking is also optimized for in turbofan..